recirq/otoc/loschmidt/__init__.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from functools import lru_cache
+from typing import Optional
+
+from cirq.protocols.json_serialization import ObjectFactory, DEFAULT_RESOLVERS
+from .tilted_square_lattice import (
+    TiltedSquareLatticeLoschmidtSpec,
+)
+
+
+@lru_cache()
+def _resolve_json(cirq_type: str) -> Optional[ObjectFactory]:
```

Comment:
I feel like I missed a change somewhere, since this is likely not the first time this snippet has appeared.
However, what is this?  I am confused why we need this.  Maybe a code comment would be good to explain what this is for.

Reply:
this is how we register modules for json deserialization; done after the separation of `cirq_google` and friends. You can see we add it to cirq's `DEFAULT_RESOLVERS` global variable

Reply:
added a docstring